.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: ruby
+rvm:
+  - 2.2.3
+  - 2.0.0
```

Comment:
Also, you want to specify the minor version, not the teeny one. Ruby is SemVer these days.